forum/qa-plugin/q2a-change-username-limit/q2a-changeusernamelimit-admin.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace CodersCommunity;
+
+class q2a_changeusernamelimit_admin
+{
+    public function init_queries()
+    {
+        $isActive = qa_opt('changeusernamelimit');
+        $result = null;
+
+        if (1 === $isActive) {
```

Comment:
Jeśli jest tu niejawne rzutowanie `1` do `true` po lewej stronie, to czy nie może być po prostu `true`? Tak będzie IMO czytelniej.